### PR TITLE
Added HBase Prometheus Dashboards

### PR DESCRIPTION
**Pull request by algchoo**
* added hbase prometheus dashboard json
* added hbase operations prometheus dashboard json
* included screenshots
* updated readme and metadata

### CODE REVIEW

File: dashboards/hbase/README.md [hunk 1]
```diff
@@ -13,4 +13,14 @@
...
\ No newline at end of file
+|This dashboard has charts for viewing Hbase Operations when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/hbase#monitored-metrics), including `Slow Get Operations`, `Slow Append Operations`, `Slow Delete Operations`, `Slow Put Operations`, `Slow Get Operations`, `Get Operations 99 Percentile`, `Append Operations 99 Percentile`, `Delete Operations 99 Percentile`, `Put Operations 99 Percentile`, `Increment Operations 99 Percentile`, and `Replay Operations 99 Percentile`. |
+
+|Hbase Prometheus Overview|
+|:------------------|
+|Filename: [hbase-prometheus-overview.json](hbase-prometheus-overview.json)|
+|This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including `Region Servers`, `Dead Region Servers`, `Read Requests`, and `Write Requests`. |
+
+|Hbase Prometheus Operations Overview|
```
Comment (varun-c) [inline dashboards/hbase/README.md:23]:
```suggestion
|Hbase Operations Prometheus Overview|
```
Based on the preview image, the name of the dashboard should be different.

File: dashboards/hbase/README.md [hunk 2]
```diff
@@ -13,4 +13,14 @@
...
\ No newline at end of file
+|This dashboard has charts for viewing Hbase Operations when monitored by [Google's Ops Agent](https://cloud.google.com/stackdriver/docs/solutions/agents/ops-agent/third-party/hbase#monitored-metrics), including `Slow Get Operations`, `Slow Append Operations`, `Slow Delete Operations`, `Slow Put Operations`, `Slow Get Operations`, `Get Operations 99 Percentile`, `Append Operations 99 Percentile`, `Delete Operations 99 Percentile`, `Put Operations 99 Percentile`, `Increment Operations 99 Percentile`, and `Replay Operations 99 Percentile`. |
+
+|Hbase Prometheus Overview|
+|:------------------|
+|Filename: [hbase-prometheus-overview.json](hbase-prometheus-overview.json)|
+|This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including `Region Servers`, `Dead Region Servers`, `Read Requests`, and `Write Requests`. |
```
Comment (varun-c) [inline dashboards/hbase/README.md:21]: https://github.com/prometheus/jmx_exporter <-- Is this the correct exporter that exposes Hbase prometheus metrics?

File: dashboards/hbase/README.md [hunk 3]
```diff
@@ -13,4 +13,14 @@
...
+
+|Hbase Prometheus Operations Overview|
+|:------------------|
+|Filename: [hbase-operations-prometheus-overview.json](hbase-operations-prometheus-overview.json)|
+|This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including `Slow Get Operations`, `Slow Append Operations`, `Slow Delete Operations`, `Slow Put Operations`, `Slow Get Operations`, `Get Operations 99 Percentile`, `Append Operations 99 Percentile`, `Delete Operations 99 Percentile`, `Put Operations 99 Percentile`, `Increment Operations 99 Percentile`, and `Replay Operations 99 Percentile`. |
```
Comment (varun-c) [inline dashboards/hbase/README.md:26]: 1. https://github.com/prometheus/jmx_exporter <-- Is this the correct exporter that exposes Hbase prometheus metrics?
2. The description has `Slow Get Operations` twice. The 2nd occurrence should be `Slow Increment Operations`
3. Replace all '99' with '99th' to be consistent with the chart titles.

File: dashboards/hbase/hbase-operations-prometheus-overview.json [hunk 4]
```diff
@@ -0,0 +1,317 @@
...
+          }
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 16
+      },
+      {
+        "height": 4,
+        "widget": {
+          "title": "Replay 99th Percentile",
```
Comment (varun-c) [inline dashboards/hbase/hbase-operations-prometheus-overview.json:290]: Should this title be "Replay Operations 99th Percentile" to be consistent with the other charts and the description?

File: dashboards/hbase/hbase-prometheus-overview.json [hunk 5]
```diff
@@ -0,0 +1,123 @@
+{
+  "category": "CUSTOM",
+  "displayName": "HBase Prometheus Overview",
+  "etag": "60a87bf42336cdeeaeb37d0a1d8de1f3",
```
Comment (varun-c) [inline dashboards/hbase/hbase-prometheus-overview.json:4]: This should be removed.

File: dashboards/hbase/hbase-prometheus-overview.json [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+            }
+          }
+        },
+        "width": 6,
+        "xPos": 6,
+        "yPos": 4
+      }
+    ]
+  },
+  "name": "projects/685351563540/dashboards/37c44280-771f-42a8-8335-a3c5b5756117"
```
Comment (varun-c) [inline dashboards/hbase/hbase-prometheus-overview.json:122]: This should be removed

File: dashboards/hbase/metadata.yaml [hunk 7]
```diff
@@ -15,3 +15,19 @@ sample_dashboards:
     related_integrations:
       - id: hbase
         platform: GCE
+  -
+    category: HBase
+    id: hbase-prometheus-overview
+    display_name: Hbase Prometheus Overview
+    description: "This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including Region Servers, Dead Region Servers, Read Requests, and Write Requests."
```
Comment (varun-c) [inline dashboards/hbase/metadata.yaml:22]: Same comment as before about the exporter used for scraping Hbase metrics. Is it https://github.com/prometheus/jmx_exporter?

File: dashboards/hbase/metadata.yaml [hunk 8]
```diff
@@ -15,3 +15,19 @@ sample_dashboards:
...
+    related_integrations:
+      - id: hbase
+        platform: GKE
+  -
+    category: HBase
+    id: hbase-operations-prometheus-overview
+    display_name: Hbase Operations Prometheus Overview
+    description: "This dashboard is based on metrics exposed by an [exporter](https://github.com/prometheus/jmx_exporter), including Slow Get Operations, Slow Append Operations, Slow Delete Operations, Slow Put Operations, Slow Get Operations, Get Operations 99 Percentile, Append Operations 99 Percentile, Delete Operations 99 Percentile, Put Operations 99 Percentile, Increment Operations 99 Percentile, and Replay Operations 99 Percentile."
```
Comment (varun-c) [inline dashboards/hbase/metadata.yaml:30]: Similar comment here as in the README.md file.

**Review by varun-c — CHANGES_REQUESTED** [review]

File: dashboards/hbase/metadata.yaml (hunk 7 above)
Comment (algchoo) [reply dashboards/hbase/metadata.yaml:22]: That is correct.

File: dashboards/hbase/hbase-operations-prometheus-overview.json (hunk 4 above)
Comment (algchoo) [reply dashboards/hbase/hbase-operations-prometheus-overview.json]: I believe so, I've updated it.

File: dashboards/hbase/README.md (hunk 3 above)
Comment (algchoo) [reply dashboards/hbase/README.md]: The exporter is correct, I've made changes to resolve the remaining inconsistencies, thanks for catching those!

File: dashboards/hbase/README.md (hunk 2 above)
Comment (algchoo) [reply dashboards/hbase/README.md:21]: That is the correct exporter.

File: dashboards/hbase/metadata.yaml (hunk 8 above)
Comment (algchoo) [reply dashboards/hbase/metadata.yaml:30]: This is the correct exporter

**Review by EvanSimpson — APPROVED** [review]

File: dashboards/hbase/metadata.yaml (hunk 8 above)
Comment (varun-c) [inline dashboards/hbase/metadata.yaml:30]: The description here should also be updated as done in the README.md file

**Review by varun-c — APPROVED** [review]

**Pull request merged**